src/app/recipe/new/_components/IngredientListItem.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+
+type Props = {
+  title: string;
+  url?: string;
```

Comment:
urlがない場合が今のところはないので（UI変更や仕様変更は考えない）必須にしてしまって良いと思います！
呼び出す側は適当に"/"指定しておく的な

```suggestion
  url: string;
```